supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobExecutionActor.scala

```diff
@@ -95,4 +98,12 @@ case class JesJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   }
 
   override def abort(): Unit = {}
+
+  // Supervision strategy: if the JABJEA throws an exception, stop the actor and fail the job.
+  def jobFailingDecider: Decider = {
+    case e: Exception =>
+      completionPromise.failure(new RuntimeException("JesAsyncBackendJobExecutionActor failed and didn't catch its exception.", e))
```

Comment:
I think `tryFailure` is better, on the off chance it completes and then throws a tidy-up exception somehow.

Changed!